### PR TITLE
Making RG optional in check policy gate, and task friendly name changed.

### PR DESCRIPTION
**Pull request by divman25**
1. Renaming the task (friendly name) to "Check Azure Policy Compliance"
2. Making the RG field optional, if the user wants to invoke the policy check across RGs in the subscription.

image

### CODE REVIEW

File: Tasks/AzurePolicyV0/task.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "version": {
     "Major": 0,
     "Minor": 0,
-    "Patch": 1
```
Comment (bansalaseem) [inline Tasks/AzurePolicyV0/task.json]: change it to 151

File: Tasks/AzurePolicyV0/task.json [hunk 2]
```diff
@@ -69,14 +68,14 @@
       "resultTemplate": "{ \"Value\" : \"{{{id}}}\", \"DisplayValue\" : \"{{{name}}}\" }"
     }
   ],
-  "instanceNameFormat": "Azure Policy compliance assessment for $(ResourceGroupName)",
+  "instanceNameFormat": "Azure Policy compliance assessment for $(ConnectedServiceName):$(ResourceGroupName)",
```
Comment (bansalaseem) [inline Tasks/AzurePolicyV0/task.json]: what will be the name of ResourceGroupName is empty?

**Comment by bansalaseem** [comment]
Please port the change to M150 as well.

File: Tasks/AzurePolicyV0/task.json (hunk 2 above)
Comment (divman25) [reply Tasks/AzurePolicyV0/task.json]: ConnectedServiceName will be the subscriptionID, if no RG is selected you'll see SubscriptionId: here.

File: Tasks/AzurePolicyV0/task.json (hunk 1 above)
Comment (divman25) [reply Tasks/AzurePolicyV0/task.json]: Done

**Pull request merged**